SECP-service/src/main/resources/assets/app/js/chat/chat-service.js

```diff
@@ -1,10 +1,98 @@
 'use strict';
 
 angular.module('SECP')
-  .factory('Chat', ['$resource', function ($resource) {
-    return $resource('SECP/chats/:id', {}, {
-      'query': { method: 'GET', isArray: true},
-      'get': { method: 'GET'},
-      'update': { method: 'PUT'}
-    });
-  }]);
+  .factory('Chat', function() {
+    // factory returns an object
+    // you can run some code before
+
+    return {
+        getMessages : function() {
```

Comment:
This structure looks good. Do you think for the time stamp it should be the time the message was sent or should it be "xx mins ago"

Reply:
Yes, it should be timestamp. We would be getting that from the server and then we can use a library to display the time on the page.

Reply:
Sounds good. That’s what I was thinking as well